src/Entity/EntityIdValue.php

```diff
@@ -67,7 +60,7 @@ public function unserialize( $serialized ) {
 		try {
 			$entityId = LegacyIdInterpreter::newIdFromTypeAndNumber( $entityType, $numericId );
 		} catch ( InvalidArgumentException $ex ) {
-			throw new IllegalValueException( 'Invalid EntityIdValue serialization.' );
+			throw new IllegalValueException( 'Invalid EntityIdValue serialization.', 0, $ex );
```

Comment:
FTR: I've merged the other PR but seriously, I think it has taken more effort to make that PR, get it reviewed and now rebase this PR, than the distraction caused by sneaking in this particular change in this PR.